src/main/java/com/direwolf20/buildinggadgets/common/config/fieldmap/FieldMapper.java

```diff
@@ -18,7 +18,7 @@
     /*@SuppressWarnings("unchecked")
     public static final FieldMapper<ImmutableList, String[]> BLOCK_LIST_MAPPER = of(
             (list) -> ((ImmutableList<Block>)list).stream().map((b) -> Objects.requireNonNull(b.getRegistryName()).toString()).toArray(String[]::new),
-            (strings) -> Stream.of(strings).map(ResourceLocation::new).map(ForgeRegistries.BLOCKS::getValue).collect(ImmutableList.toImmutableList()),
+            (strings) -> Stream.of(strings).map(ResourceLocation::new).map(ForgeRegistries.BLOCKS::getValue).collectCoordinates(ImmutableList.toImmutableList()),
```

Comment:
please revert that XD

Reply:
Oh wait IntelliJ renames everything that has the same name?
Sorry about that

Reply:
you need to deactivate the "search in comments and Strings" checkbox, to keep it from doing that... Has the negative side-effect of them being ignored. So Iguess one can't do much about it.